MaxText/layers/models.py

```diff
@@ -804,4 +802,3 @@ def __call__(
         image_embeddings=image_embeddings,
     )
     return logits
-  
```

Comment:
Same question around removing the new line at the end of file

Reply:
Again just default running of `pyink` no special setup